assets/docs/adityakane2001/models/regnety200mf_classification/1.md

```diff
@@ -0,0 +1,42 @@
+# Module adityakane2001/regnety200mf_classification/1
+
+RegNetY 200MF image classifier pretrained on ImageNet-1k.
+
+<!-- asset-path: https://storage.googleapis.com/ak-regnety-savedmodels/tars/regnety200mf_classification.tar.gz  -->
+<!-- task: image-classification -->
+<!-- network-architecture: regnety200mf_classification -->
+<!-- format: saved_model_2 -->
+<!-- fine-tunable: false -->
+<!-- license: apache-2.0 -->
+<!-- language: en -->
+<!-- colab:    -->
+
+## Overview
+
+This model is the implementation of  RegNetY<sup>[1]</sup>  in TensorFlow 2.5.
+
+## Using this model
+
+This model is a pretrained classifier on ImageNet-1k. This means that you can directly classify images over 1000 predefined classes.
+
+```python
+model = tf.keras.Sequential([
+    hub.KerasLayer("https://tfhub.dev/adityakane2001/regnety200mf_classification/1")
+])
+predictions = model.predict(images) 
+
+# `images` must be a Tensor of the shape 
+# (batch_size, 224, 224, 3)
+# or must be generator/ iterator/ tf.data.Dataset 
+# furnishing Tensors of given dimensions. 
+```
```

Comment:
We might want to also note that the pixel range of the input images should be [0, 255]. If you also want to add a note on running on CPU (with Group Convs) I am okay with that.

Reply:
> ```
# `images` must be a Tensor of the shape 
# (batch_size, 224, 224, 3)
```

Actually this is incorrect. Users can pass images of any size, as we resize the images inside the model. I've made that change in the next commit. I've also added the grouped conv and [0,255] part.